prow/istio-presubmit.sh

```diff
@@ -65,24 +65,17 @@ fi
 cd $ROOT
 
 echo 'Running Unit Tests'
-bazel test --test_output=all //...
+time bazel test --test_output=all //...
 
 # ensure that source remains go buildable
-${ROOT}/bin/init.sh
+SKIP_BUILD=1 ${ROOT}/bin/init.sh
```

Comment:
I don’t quite understand how you can push artefacts without building ?

Reply:
The previous line `bazel test --test_output=all //...`  Build and unit tests the code"

Reply:
then (re)building after is a noop anyway ? I didn't know that the test target builds everything (that seems off, why would test build non test binaries)